package.json

```diff
@@ -90,6 +91,7 @@
     "react-dom": "16.10.0",
     "react-emotion": "9.2.12",
     "react-hot-loader": "4.12.11",
+    "react-i18next": "10.13.1",
```

Comment:
Please why did you decided to use "react-i18next"?.. it's just matter of curiosity..I have never used this library before.